Examples/php/cpointer/runme.php

```diff
@@ -1,40 +1,38 @@
 <?php
 
-	require "example.php";
-
 	# First create some objects using the pointer library.
 
 	print "Testing the pointer library\n";
 
-	$a = example::new_intp();
-	$b = example::new_intp();
-	$c = example::new_intp();
-	example::intp_assign($a,37);
-	example::intp_assign($b,42);
+	$a = new_intp();
+	$b = new_intp();
+	$c = new_intp();
+	intp_assign($a,37);
+	intp_assign($b,42);
 
 	print "	a = $a\n";
 	print "	b = $b\n";
 	print "	c = $c\n";
 
 	# Call the add() function wuth some pointers
-	example::add($a,$b,$c);
+	add($a,$b,$c);
 
 	# Now get the result
-	$r = example::intp_value($c);
+	$r = intp_value($c);
 
 	print "	37 + 42 = $r\n";
 
 	# Clean up the pointers
-	example::delete_intp($a);
-	example::delete_intp($b);
-	example::delete_intp($c);
+	delete_intp($a);
+	delete_intp($b);
+	delete_intp($c);
 
 	# Now try the typemap library
 	# This should be much easier. Now how it is no longer
 	# necessary to manufacture pointers.
 
 	print "Trying the typemap library\n";
-	$r = example::sub(37,42);
+	$r = sub(37,42);
```

Comment:
At the moment, the `delete_intp` is the flat wrapper layer and the `example::delete_intp` is the class wrapper around that.  If we were starting from scratch it probably would make more sense to wrap non-class function C/C++ function `intp_value` as a non-class PHP function, rather than placing it in this `example` class, especially now PHP has added namespaces, though I think we probably need to have an option to generate the `example` class for compatibility.  With current wrappers, I'm not sure if you can just globally replace `example::x` with `x`, or if there will be issues when objects are passed and/or returned, since they are handled as resources by this layer.